src/components/RegionMap/MetroAreaMap.tsx

```diff
@@ -0,0 +1,161 @@
+import React, { useState } from 'react';
+import { Link } from 'react-router-dom';
+import { uniq } from 'lodash';
+import { ComposableMap, Geographies } from 'react-simple-maps';
+import COUNTIES_JSON from 'components/Map/data/counties-small.json';
+import { geoBounds, geoCentroid, geoDistance } from 'd3-geo';
+import * as topojson from 'topojson-client';
+import { MetroArea } from 'common/regions';
+import * as Styles from './MetroAreaMap.style';
+import regions from 'common/regions';
+import { LocationSummariesByFIPS } from 'common/location_summaries';
+import ReactTooltip from 'react-tooltip';
+
+const MetroAreaMap: React.FC<{
+  height?: number;
+  width?: number;
+  metroArea: MetroArea;
+}> = ({ height = 600, width = 800, metroArea }) => {
+  const [tooltipContent, setTooltipContent] = useState('');
+
+  const countyFipsList = metroArea.counties.map(county => county.fipsCode);
+  const countiesTopoJson = buildCountyGeometries(countyFipsList);
+  const projectionConfig = getProjectionConfig(countiesTopoJson, width, height);
+
+  const stateFipsList = getStateFipsList(countyFipsList);
+  const statesTopoJson = buildStateGeometries(stateFipsList);
+
+  const onMouseEnter = (fipsCode: string) => {
+    setTooltipContent(getNameByFips(fipsCode));
+  };
+
+  const onMouseLeave = () => setTooltipContent('');
+
+  return (
+    <Styles.MapContainer>
+      <ComposableMap
+        projection={'geoConicEqualArea'}
+        height={height}
+        width={width}
+        projectionConfig={projectionConfig}
+        data-tip=""
+      >
+        <Geographies key="states" geography={statesTopoJson}>
+          {({ geographies }) =>
+            geographies.map(geo => (
+              <Link
+                key={geo.rsmKey}
+                to={getRelativeUrlByFips(geo.id)}
```

Comment:
thoughts on loading the region once? This strategy kinda depends on all of the regions being there, but they should be, right?
```
geo => {
  const region = regions.getRegionsByFips(geo.id);
  assert(region);
  return (
   <Geographies> ... (then you don't need the helper functions)
  )
}

```